src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -361,6 +361,11 @@ public static class Popup
 			/// This is mainly useful for debugging purposes, we do not recommend using this in production code.
 			/// </summary>
 			public static bool PreventLightDismissOnWindowDeactivated { get; set; }
+
+			/// <summary>
+			/// By default, popups are constrained by the visible bounds.
+			/// </summary>
+			public static bool ConstrainByVisibleBounds { get; set; } = true;
```

Comment:
Why is this configurable? I'm very much against adding feature config options unless we desperately need to.